python/src/bind_zmq.cpp

```diff
@@ -0,0 +1,87 @@
+#include "communication_interfaces_bindings.hpp"
+
+#include <communication_interfaces/sockets/ZMQPublisherSubscriber.hpp>
+
+using namespace communication_interfaces::sockets;
+
+class PyZMQContext {
```

Comment:
I hope you are happy with this workaround for the zmq context. Basically, instead of using `pyzmq` to create your context, we just use this one and everything gets simpler.

Reply:
yes I think that's the best solution based on what we discussed. Despite it not being possible to construct a `cppzmq` context from an existing `pyzmq` context, having this binding for a `cppzmq` context as a python object still allows the context to be persisted between configs, which is the main use case.